type_lens/type_view.py

```diff
@@ -43,13 +43,13 @@ def __init__(self, annotation: T) -> None:
         unwrapped, metadata, wrappers = unwrap_annotation(annotation)
         origin = get_origin(unwrapped)
 
-        args = () if origin is abc.Callable else get_args(unwrapped)
+        args: tuple[Any, ...] = () if origin is abc.Callable else get_args(unwrapped)
```

Comment:
Ran into this attempting to use `args` directly somewhere, gets type errors because the inferred type is `tuple[()] | tuple[Any, ...]`, which fails if you do `args[0]`.

In reality, it really is `tuple[Any, ...]` which is just a wider type with less constraint, which allows unchecked index access, it seems more correct